pyvista/core/filters/poly_data.py

```diff
@@ -2587,6 +2600,15 @@ def extrude(self, vector, capping=False, inplace=False, progress_bar=False):
         >>> mesh.plot(line_width=5, show_edges=True)
 
         """
+        if capping is None:  # pragma: no cover
```

Comment:
@adeak Sorry. I guess I didn't communicate it well enough to you. I agree 100% about the default argument. My point was that since I don't like to use `# pragma: no cover`, is there any way to test this line. I suggested using `GetCapping()` to check the value as an example for that. I don't care if it turns out to be True or False. I guess the cover problem can also be solved with a test that does nothing and only executes the method :)

Reply:
Ah, OK, thanks for clarifying. This is now done following the same suggestion from @akaszynski

Reply:
I worked hard to create a long English sentence and it was already reflected 🥲.